packages/data/tests/unit/adapters/dimensions/bard-test.js

```diff
@@ -66,7 +67,11 @@ module('Unit | Adapter | Dimensions | Bard', function(hooks) {
 
     //Load metadata
     Server.map(metadataRoutes);
-    return this.owner.lookup('service:bard-metadata').loadMetadata();
+    metadataRoutes.bind(Server)(1);
+    return all([
```

Comment:
Do we need RSVP.all instead of Promise.all?

Reply:
yeah didn't work right with promise.all.